Sources/BraintreeCoreSwift/BTAnalyticsEvent.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+/// Encapsulates a single analytics event
+struct BTAnalyticsEvent {
+
+    var eventName: String
+    var timestamp: Double
+
+    var description: String {
+        "\(eventName) at \(timestamp)"
+    }
+
+    /// Event serialized to JSON
+    var json: [String: Any] {
+        [
+            "eventName": eventName,
```

Comment:
I notice this changed from `kind` to `eventName`. I think we want to keep parity with existing Arachne implementations. (Until we're ready to go full FPTI).

Reply:
Some background here - we'll be forwarding Arachne events to FPTI at some point, and the less ways we have of doing something on Arachne, the less logic we'll need to implement later to make sure we're handling all the varied structure of analytics to FPTI.

Reply:
Ah, you're right. I did not resize this actually maps to the event: `"analytics":[{"kind":"ios.started.client-key","timestamp":1667492509855}]}}`. What do we think of keeping the eventName for the internal properties and reverting the JSON back to kind? So:
```
var json: String: Any {
    [
        "kind": eventName,
       "timestamp": timestamp
    ]
}
```

Reply:
This was updated with the above in: https://github.com/braintree/braintree_ios/pull/887/commits/d78d7b0e9ef97f846c0f6f36aa52447615c24615